client/src/components/Profiles.js

```diff
@@ -81,9 +81,8 @@ const Profiles = () => {
       isMatch(profile.email) ||
       (profile.address && isMatch(profile.address)) ||
       (profile.phone_number && isMatch(profile.phone_number)) ||
-      (profile.occupation && isMatch(profile.occupation))(
-        profile.group_id && isMatch(profile.group_id)
-      )
+      (profile.occupation && isMatch(profile.occupation)) ||
+      (profile.group_id && isMatch(profile.group_id))
```

Comment:
Just remove the match check on `group_id` please since this isn't a valid profile field. It's left over from an earlier version which assumed there would only be one group.